.travis.yml

```diff
@@ -9,7 +9,7 @@ cache:
     - $HOME/.sbt
     - $HOME/.ivy2
 script:
-  - sbt "set parallelExecution in Test := false" clean coverage doc assembly
+  - sbt clean scapegoat doc coverage package
```

Comment:
`sbt clean scapegoat doc coverage test coverageReport` yields this error

```
[info] Statement coverage.: 73.42%
[info] Branch coverage....: 70.45%
[info] Coverage reports completed
[error] Coverage is below minimum [73.42% < 75.0%]
```

Could lower the coverage in this diff to 70%? Thanks a ton.